admin-dev/themes/new-theme/js/components/grid/extension/link-row-action-extension.js

```diff
@@ -71,7 +71,7 @@ export default class LinkRowActionExtension {
          * Only search for cells with non clickable contents to avoid conflicts with
          * previous cell behaviour (action, toggle, ...)
          */
-        const clickableCells = $('td.data-type, td.identifier-type, td.badge-type, td.position-type', $parentRow)
+        const clickableCells = $('td.data-type, td.identifier-type:not(:has(.md-checkbox)), td.badge-type, td.position-type', $parentRow)
```

Comment:
What about radio? text? etc :thinking: Are we sure, `.md-checkbox` is enough? what if a thrid part module use a custom css class?

Could we test something more generic?
```suggestion
        const clickableCells = $('td.data-type, td.identifier-type:not(:has(input)), td.badge-type, td.position-type', $parentRow)
```